src/main/java/com/scottlogic/competitionEngine/application/user/UserServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package com.scottlogic.competitionEngine.application.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserServiceImpl {
+
+    @Autowired
+    public UserServiceImpl() {}
+
+    @CrossOrigin (origins = "http://localhost:3000")
+    @GetMapping("/authorise")
+    public String authoriseToken() {
```

Comment:
this should not be authorise - we will get spring security to do that for us, this is just /users/current

Also lets leave CORS out until we actually implement security